Assets/Scripts/Player/PlayerHandDetection.cs

```diff
@@ -3,19 +3,32 @@
 using UnityEngine;
 
 public class PlayerHandDetection : MonoBehaviour {
+	[SerializeField] private GameObject interactArrow;
```

Comment:
Instead of letting the playerhand monitor the interact arrow, would it be better to let the interactable item handle it themselves?